test/playback-watcher.test.js

```diff
@@ -1006,6 +1006,12 @@ QUnit.test('jumps to buffered content if seeking just before', function(assert)
     currentTime: () => currentTime,
     buffered: () => buffered
   };
+
+  Object.assign(playbackWatcher.masterPlaylistController_.sourceUpdater_, {
```

Comment:
had to update this test to pass videoBuffered, other than that all tests continue to work.